src/main/scala/com/fulcrumgenomics/illumina/SampleSheet.scala

```diff
@@ -116,8 +127,8 @@ object SampleSheet {
 
   /** Creates a sample from the given row data. */
   protected def makeSample(sampleOrdinal: Int, sampleDatum: Map[String, String]): Sample = {
-    val sampleName: String          = SampleSheet.getStringField(sampleDatum, SampleName)    getOrElse (throw new IllegalArgumentException(s"Missing: $SampleName"))
-    val sampleId: String            = SampleSheet.getStringField(sampleDatum, SampleId)      getOrElse (throw new IllegalArgumentException(s"Missing: $SampleId"))
+    val sampleName: String          = SampleSheet.getStringField(sampleDatum, SampleName)    getOrElse (throw new IllegalArgumentException(s"Sample at ordinal #$sampleOrdinal is missing: $SampleName"))
+    val sampleId: String            = SampleSheet.getStringField(sampleDatum, SampleId)      getOrElse (throw new IllegalArgumentException(s"Sample at ordinal #$sampleOrdinal is missing: $SampleId"))
```

Comment:
Similar to above. Before, the "comma-only" lines just gave the error message "Missing: Sample_Name", which was a bit confusing.